ui/lib/helper.ts

```diff
@@ -229,3 +229,5 @@ export const permissionFormFields: PermissionInfo[] = [
     description: "Provides access to billing settings and invoices",
   },
 ];
+
+export type ProviderType = "aws" | "gcp" | "azure" | "kubernetes" | "m365";
```

Comment:
Could you please revert this change? I believe it's not part of the Lighthouse scope, thank you!

For more context: I'm already moving this in another PR. We try to keep the scope of each PR as limited as possible, so it's clear exactly what was done and where.